ethiopian_date/ethiopian_date/ethiopian_date.py

```diff
@@ -221,4 +221,4 @@ def to_ethiopian(cls, year, month, date):
         order = [0, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 1, 2, 3, 4]
         ethiopian_month = order[m]
 
-        return datetime.date(ethiopian_year, ethiopian_month, ethiopian_date)
+        return '{}-{}-{}'.format(ethiopian_year, ethiopian_month, ethiopian_date)
```

Comment:
I think it might be better to return this as a tuple `(ethiopian_year, ethiopian_month, ethiopian_date)`, so that the caller can decide what do do with it themselves.

Reply:
Updated